tests/analyses/reaching_definitions/test_subject.py

```diff
@@ -19,8 +17,8 @@ def test_can_be_instantiated_with_a_function(_):
     function = _a_mock_function(0x42, 'function_name')
     subject = Subject(function)
 
-    nose.tools.assert_equals(subject.content, function)
-    nose.tools.assert_equals(subject.type, SubjectType.Function)
+    assert (subject.content == function)
+    assert (subject.type == SubjectType.Function)
```

Comment:
Remove the surrounding parentheses please. It should be `assert subject.type == SubjectType.Function`.